### DISCUSSION TITLE
Create order issue with no reason

### DISCUSSION THREAD

**Discussion by FloUz-ai** [ProjectOpenSea/seaport#1006] [Q&A]
Hi guys, I have this code snippet:

```
const { executeAllActions } = await seaport0x4.createOrder(
    {
      offer: [
        {
          itemType: ItemType.ERC721,
          token: "0x2a22c4b53585e12ca6b2c180f6ac1a3b1f6c77f5",
          identifier: "1",
          amount: "1",
          endAmount: "1",
        },
      ],
      consideration: [
        {
          amount: "50000000000000000",
          token: "0x0000000000000000000000000000000000000000",
          recipient: account0x4,
        },
      ],
      // fees: [
      //   {
      //     recipient: "account0x4",
      //     basisPoints: 250,
      //   },
      // ],
    },
    account0x4
  );

  const order = await executeAllActions();
```
It works for main of my listings but for any reason, I have on token with which it doesnt want to create any order. It ends with an error which does not say much : 

```  
reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0x2A22C4b53585E12CA6B2C180f6ac1a3B1f6c77f5',
    data: '0xe985e9c500000000000000000000000004b6669196fe2e77b6a7c4527f11b923efe7641a00000000000000000000000000000000006c3852cbef3e08e8df289169ede581',
    accessList: null
  },
```

Anyone already had this kind of error? I also have a code -32000

The weirdest thing is that it is working when I try to create an order on opensea.. it seems something is missing... but what??

**Comment by dhanjani** [comment]


**Reply by FloUz-ai** [reply]
I was thinking the same, as it is not triggered on my app.. but The contract is created from ERC721 from openZeppelin, And I have the same with one from the ERC721A..

**Comment by ryanio** [comment]
Your token is reverting on `isApprovedForAll`, is a token actually deployed at that expected address of `0x2A22C4b53585E12CA6B2C180f6ac1a3B1f6c77f5`
